config/formats.js

```diff
@@ -195,6 +195,7 @@ let Formats = [
 			battle: 3,
 		},
 		ruleset: ['Standard GBU'],
+		banlist: ['Copperajah-Gmax', 'Duraludon-Gmax', 'Garbodor-Gmax', 'Gengar-Gmax', 'Machamp-Gmax'],
 		minSourceGen: 8,
 		onValidateSet(set) {
```

Comment:
You still need to delete this, right?

Reply:
i definitely deleted those idk why they are still there. weird